src/System.CommandLine/Parsing/ParseOperation.cs

```diff
@@ -243,7 +235,7 @@ private void ParseRemainingTokens()
                 }
                 else if (foundEndOfArguments)
                 {
-                    AddCurrentTokenToUnparsed();
+                    AddCurrentTokenToUnmatched();
```

Comment:
I don't think this is the right behavior. This should probably not doing anything special at this point.

Reply:
Yeah, you are right. The whole method become useless. I removed it.